src/gui/MainWindow.cpp

```diff
@@ -295,10 +295,15 @@ MainWindow::MainWindow()
     new QShortcut(Qt::CTRL + Qt::Key_M, this, SLOT(minimizeOrHide()));
     new QShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_M, this, SLOT(hideWindow()));
     // Control database tabs
-    new QShortcut(Qt::CTRL + Qt::Key_Tab, this, SLOT(selectNextDatabaseTab()));
-    new QShortcut(Qt::CTRL + Qt::Key_PageDown, this, SLOT(selectNextDatabaseTab()));
-    new QShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_Tab, this, SLOT(selectPreviousDatabaseTab()));
-    new QShortcut(Qt::CTRL + Qt::Key_PageUp, this, SLOT(selectPreviousDatabaseTab()));
+    // Ctrl+Tab is broken on Mac, so use Alt (i.e. the Option key) - https://bugreports.qt.io/browse/QTBUG-8596
+    auto dbTabModifier2 = Qt::CTRL;
+#ifdef Q_OS_DARWIN
+    dbTabModifier2 = Qt::ALT;
```

Comment:
This should be `Qt::META` instead which will get converted to the Ctrl key on Mac.

Reply:
The reason why we can't use Ctrl is noted in the comment above:
```
// Ctrl+Tab is broken on Mac, so use Alt (i.e. the Option key) - https://bugreports.qt.io/browse/QTBUG-8596
```

Reply:
Good point, but you are converting the working Cmd+Pgup/dn to Alt as well. That is not a desirable change.

Reply:
I did mention this in the description above as well. I will change it back.

> I also changed Ctrl+PageDown for consistency, but I can change that back. I can also change the variable name. I wanted to quickly create this PR with the gist of the proposed change in order to get feedback.

I am also ok with closing the PR if you want to take care of this on your own, now that I have raised the issue. There are so many minor things that are wrong with my changes (e.g. `dbTabModifier2` is a bad variable name). So feel free to close my PR and address this problem yourself. You may even be able to get Ctrl+Tab working. I didn't understand the bug well enough to attempt it on my own, but maybe you do. Thanks.

Reply:
This is all part of the process no worries